addon/components/mdc-tab-bar-scroller.js

```diff
@@ -99,45 +101,47 @@ export default Component.extend(MDCComponent, {
-      removeClassFromBackIndicator: className => run(() => get(this, 'backIndicatorClasses').removeObject(className)),
+      addClassToBackIndicator: (className) => run(() => get(this, 'backIndicatorClasses').addObject(className)),
+      removeClassFromBackIndicator: (className) => run(() => get(this, 'backIndicatorClasses').removeObject(className)),
       isRTL: () =>
         !!this &&
         !get(this, 'isDestroyed') &&
         !!get(this, 'element') &&
         getComputedStyle(get(this, 'element')).getPropertyValue('direction') === 'rtl',
-      registerBackIndicatorClickHandler: handler =>
+      registerBackIndicatorClickHandler: (handler) =>
         get(this, 'backIndicatorElement').addEventListener('click', handler),
-      deregisterBackIndicatorClickHandler: handler =>
+      deregisterBackIndicatorClickHandler: (handler) =>
         get(this, 'backIndicatorElement').removeEventListener('click', handler),
-      registerForwardIndicatorClickHandler: handler =>
+      registerForwardIndicatorClickHandler: (handler) =>
         get(this, 'forwardIndicatorElement').addEventListener('click', handler),
-      deregisterForwardIndicatorClickHandler: handler =>
+      deregisterForwardIndicatorClickHandler: (handler) =>
         get(this, 'forwardIndicatorElement').removeEventListener('click', handler),
       registerCapturedInteractionHandler: (evt, handler) => get(this, 'element').addEventListener(evt, handler, true),
       deregisterCapturedInteractionHandler: (evt, handler) =>
         get(this, 'element').removeEventListener(evt, handler, true),
-      registerWindowResizeHandler: handler => window.addEventListener('resize', handler),
-      deregisterWindowResizeHandler: handler => window.removeEventListener('resize', handler),
+      registerWindowResizeHandler: (handler) => window.addEventListener('resize', handler),
+      deregisterWindowResizeHandler: (handler) => window.removeEventListener('resize', handler),
       getNumberOfTabs: () => get(this, 'tab-bar.tabs.length'),
-      getComputedWidthForTabAtIndex: index =>
+      getComputedWidthForTabAtIndex: (index) =>
         getComponentProperty(get(this, 'tab-bar').tabAt(index), 'computedWidth', 0),
-      getComputedLeftForTabAtIndex: index => getComponentProperty(get(this, 'tab-bar').tabAt(index), 'computedLeft', 0),
+      getComputedLeftForTabAtIndex: (index) =>
+        getComponentProperty(get(this, 'tab-bar').tabAt(index), 'computedLeft', 0),
       getOffsetWidthForScrollFrame: () => get(this, 'scrollFrameElement').offsetWidth,
       getScrollLeftForScrollFrame: () => get(this, 'scrollFrameElement').scrollLeft,
-      setScrollLeftForScrollFrame: scrollLeftAmount => (get(this, 'scrollFrameElement').scrollLeft = scrollLeftAmount),
+      setScrollLeftForScrollFrame: (scrollLeftAmount) =>
+        (get(this, 'scrollFrameElement').scrollLeft = scrollLeftAmount),
```

Comment:
Does this version of Ember not yet expect this to instead be `this.scrollFrameElement`?

Reply:
While I do think this.scrollFrameElement will work, I haven't made this change generally.